src/modules/cmdpal/Microsoft.CmdPal.Core.ViewModels/CommandBarViewModel.cs

```diff
@@ -149,6 +149,7 @@ private ContextKeybindingResult PerformCommand(CommandItemViewModel? command)
 
         if (command.HasMoreCommands)
         {
+            WeakReferenceMessenger.Default.Send<PerformCommandMessage>(new(command.Command.Model, command.Model));
```

Comment:
okay I was pretty sure that this would run afoul of the same "clicking on a nested command with more commands invokes AND opens the menu" but it would seem that I'm wrong. 

Good enough for me